src/main.cpp

```diff
@@ -313,7 +314,7 @@ int main(int argc, char** argv) {
 #endif
     CZNC::CreateInstance();
 
-    while ((iArg = getopt_long(argc, argv, "hvnrcspd:Df", g_LongOpts,
+    while ((iArg = getopt_long(argc, argv, "hvnrcs::pd:Df", g_LongOpts,
```

Comment:
`man getopt_long` says this is GNU extension. Does it work on other libc?

Reply:
I've no idea, I'm afraid :disappointed: C++ is not my forté. I guess the alternative is to introduce a wholly new argument, but at that point it'd probably be simpler just build the `<Pass>` elements by hand.